CHANGELOG.md

```diff
@@ -18,308 +18,311 @@ Bug-fixes within the same version aren't needed
 * quoting default "pathToJest" to preserve special characters, if any. - @connectdotz
 * Fix for when `branch.end.column` is `null`. [@garyking](https://github.com/garyking)
 * Fix several jest coverage issues and improve usability. - @connectdotz
+* Add "Running tests" message to the StatusBar - [@jmarceli](https://github.com/jmarceli)
 
 -->
 
 ### 3.0.1
 
-* use webpack for compilation, resulting in a much smaller extension size - stephtr
```

Comment:
Was that change (change stars to dashes) made deliberately?

Reply:
I'm sorry that was prettier autoformat in my VS Code, I will revert it in a minute.

Reply:
Done